get_study_attachments.py

```diff
@@ -0,0 +1,21 @@
+import sys
+import boto3
+import botocore
+
+BUCKET_NAME = 'mitLookit'
```

Comment:
This should be in a setting that pulls from an environment variable and has a default value for local development that isn't this real bucket name. We don't have to scrub this commit but defaults.py should have something fake and then developers can put a real value in their local.py that doesn't get committed.